applications/tari_console_wallet/src/ui/components/notification_tab.rs

```diff
@@ -0,0 +1,73 @@
+use crate::ui::{components::Component, state::AppState};
+use tari_comms::runtime::Handle;
+// use tari_p2p::auto_update::SoftwareUpdaterService;
+use tui::{
+    backend::Backend,
+    layout::{Constraint, Layout, Rect},
+    style::{Color, Modifier, Style},
+    text::{Span, Spans},
+    widgets::{Block, Borders, Paragraph, Wrap},
+    Frame,
+};
+
+pub struct NotificationTab {}
+
+impl NotificationTab {
+    pub fn new() -> Self {
+        Self {}
+    }
+
+    fn draw_notifications<B>(&mut self, f: &mut Frame<B>, area: Rect, app_state: &AppState)
+    where B: Backend {
+        let block = Block::default().borders(Borders::ALL).title(Span::styled(
+            "Update",
```

Comment:
Minor point but I'd prefer this to say notifications as well

```suggestion
            "Notifications",
```